examples/nas/classic_nas/config_nas.yml

```diff
@@ -0,0 +1,17 @@
+authorName: default
+experimentName: example_mnist
+trialConcurrency: 1
+maxExecDuration: 1h
+maxTrialNum: 10
+#choice: local, remote, pai
+trainingServicePlatform: local
+searchSpacePath: NNI_AUTO_GEN
```

Comment:
it looks no document explain NNI_AUTO_GEN, so that the file may be confusing.

Reply:
Is it possible to use the user set path also, and put auto generated search space in that folder?

Reply:
thanks, I will refactor this part.